src/bootstrap/src/core/build_steps/compile.rs

```diff
@@ -1229,9 +1230,16 @@ fn rustc_llvm_env(builder: &Builder<'_>, cargo: &mut Cargo, target: TargetSelect
     }
 }
 
+/// `RustcLink` copies all of the rlibs from the rustc build into the previous stage's sysroot.
```

Comment:
Technically, `RustcLink` doesn't care about stages. 

```suggestion
/// `RustcLink` copies all of the rlibs from `compiler` sysroot into `target_compiler` sysroot.
```

Reply:
I think very valuable to explicitly mention that it's the previous stage

Reply:
Sure. If we want to explicitly mention "previous stage", it's worth updating the field names to something like `compiler_from_previous_stage` or `previous_stage_compiler` so that the comment aligns with the type.